spi/identity-hub-store-spi/build.gradle.kts

```diff
@@ -1,3 +1,13 @@
 plugins {
     `java-library`
+    `maven-publish`
+}
+
+publishing {
+    publications {
```

Comment:
not needed at this point but should plan to include `signing` plugin for publishing to Maven Central.

Also will require tasks for including javadoc, source jars  and signing, can also be done at root level